tst/com/amazon/corretto/crypto/provider/test/EcGenTest.java

```diff
@@ -358,10 +362,13 @@ public static void assertECEquals(
 
   public static void assertECEquals(
       final String message, final ECParameterSpec expected, final ECParameterSpec actual) {
-    assertEquals(expected.getCofactor(), actual.getCofactor(), message);
-    assertEquals(expected.getOrder(), actual.getOrder(), message);
-    assertEquals(expected.getGenerator(), actual.getGenerator(), message);
-    assertEquals(expected.getCurve(), actual.getCurve(), message);
+    boolean result = false;
+    try {
+      result = TestUtil.sneakyInvoke(EC_UTILS_CLASS, "ecParameterSpecsAreEqual", expected, actual);
+    } catch (final Throwable t) {
+      throw new AssertionError(message, t);
+    }
+    assertTrue(message, result);
```

Comment:
should this be `assertTrue(result, message)`?

https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html#assertTrue-boolean-java.lang.String-

Reply:
Somehow the wrong `assertTrue` also got pulled in. Not sure how I did that. Thanks for the catch. I'll squash that patch and force push shortly.